docs/stencil-docs/installing-stencil-cli/stencil-cli-options-and-commands.md

```diff
@@ -159,19 +159,33 @@ Bundles up the theme into a structured `.zip` file, which can be uploaded to Big
 stencil bundle
 ```
 
-<a id="stencil-push"></a>
+<a id="stencil-pull"></a>
 
 ## `stencil pull`
 
-Pulls the theme configuration `config.json` file from the live store and updates the local configuration.
+Pulls the configuration from the active theme on your live store and updates your local configuration. This is useful if any theme settings have been changed within Page Builder, as it will prevent you from overwriting them with your next theme upload by first syncing them.
```

Comment:
There are some changes to `pull` in https://github.com/bigcommerce/stencil-cli/pull/646, this better describes the behavior now.